tools/distpartitioning/data_shuffle.py

```diff
@@ -822,15 +841,15 @@ def prepare_local_data(src_data, local_part_id):
         num_edges = shuffle_global_eid_offsets[local_part_id]
         node_count = len(node_data[constants.NTYPE_ID+"/"+str(local_part_id)])
         edge_count = len(edge_data[constants.ETYPE_ID+"/"+str(local_part_id)])
-        local_node_data = prepare_local_data(node_data, local_part_id)
-        local_edge_data = prepare_local_data(edge_data, local_part_id)
+        local_node_data = prepare_local_data(node_data, local_part_id, 2)
```

Comment:
as we discussed, let's assert before calling `prepare_local_data`. and remove the 3rd argument from the function.